stdlib/universe/kaufmansAMA_test.go

```diff
@@ -0,0 +1,209 @@
+					{int64(1)},
+				},
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{10.444444444444445},
+					{11.135802469135802},
+					{11.964334705075446},
+					{12.869074836153025},
+					{13.81615268675168},
+					{13.871008014588556},
+					{13.71308456353558},
+					{13.553331356741122},
+					{13.46599437575161},
+					{13.4515677602438},
+					{13.29930139347417},
+					{12.805116570729282},
+					{11.752584300922965},
+					{10.036160535131101},
+					{7.797866963961722},
+					{6.109926091089845},
+					{4.727736717272135},
+					{3.515409287373408},
+					{2.3974496040963373},
+				},
+			}},
+		},
+		{
+			name: "flux.TTime & flux.TInt input",
+			spec: &universe.KamaProcedureSpec{
+				N:       1,
+				Columns: []string{"_value"},
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TInt},
+				},
+				Data: [][]interface{}{
+					{execute.Time(12), int64(1)},
+					{execute.Time(14), int64(2)},
+					{execute.Time(14), int64(4)},
+				},
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(14), 1.4444444444444446},
+					{execute.Time(14), 2.5802469135802473},
+				},
+			}},
+		},
+		{
+			name: "flux.TTime & flux.TUInt input",
+			spec: &universe.KamaProcedureSpec{
+				N:       1,
+				Columns: []string{"_value"},
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TUInt},
+				},
+				Data: [][]interface{}{
+					{execute.Time(12), uint64(1)},
+					{execute.Time(14), uint64(2)},
+					{execute.Time(14), uint64(4)},
+				},
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(14), 1.4444444444444446},
+					{execute.Time(14), 2.5802469135802473},
+				},
+			}},
+		},
+		{
+			name: "flux.TTime & flux.TFloat input",
+			spec: &universe.KamaProcedureSpec{
+				N:       1,
+				Columns: []string{"_value"},
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(12), 1.0},
+					{execute.Time(14), 2.0},
+					{execute.Time(14), 4.0},
+				},
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(14), 1.4444444444444446},
+					{execute.Time(14), 2.5802469135802473},
+				},
+			}},
+		},
+	}
```

Comment:
You might want to add test coverage for some of the error cases too (like if a column is not a type that kama will accept).